app/views/confirm.njk

```diff
@@ -69,14 +69,20 @@
     </form>
   </div>
   <div class="govuk-grid-column-one-third payment-summary-wrap">
-    <aside class="payment-summary">
+    <aside class="payment-summary govuk-!-font-size-19">
       <h2 class="govuk-heading-m">{{ __("paymentSummary.title") }}</h2>
-      <p id="payment-description" class="govuk-body govuk-!-font-weight-bold govuk-!-font-size-39">
```

Comment:
I dont know how `govuk-!-font-size-39` ever got here (although I can see in the history it was me 😬 ) `govuk-body` sets the size to 19px so it‘s redundant.